doc/api/fs.md

```diff
@@ -1243,6 +1243,13 @@ possible exception are given to the completion callback.
 
 See also: chmod(2).
 
+```js
+fs.chmod('my_file.txt', 775, (err) => {
```

Comment:
This line has an important error that needs fixing! This code will not change the mode to 775. The `mode` argument needs to be changed.

```suggestion
fs.chmod('my_file.txt', 0o775, (err) => {
```

This is detailed in the subsequent "File modes" section.